doc/DISTUTILS.rst.txt

```diff
@@ -314,6 +391,22 @@ mimicking languages that have generic type support. By convention,
 and as required by the preprocessor, generically typed blocks are preceded
 by comment blocks that enumerate the intended string substitutions.
 
+"\*/ "on a line by itself marks the end of the variable expansion
+naming. The next line is the first line that will be repeated using
+the named rules.
+
+Named variable expansions are defined using #name=item1, item2, item3,
+..., itemN# and placed on successive lines. These variables are
+replaced in each repeat block with corresponding word. All named
+variables in the same+ repeat block must define the same number of words.
+
+In specifying the repeat rule for a named variable, item*N is short-
+hand for item, item, ..., item repeated N times. In addition,
+parenthesis in combination with \*N can be used for grouping several
+items that should be repeated. Thus, #name=(item1, item2)*4# is
+equivalent to #name=item1, item2, item1, item2, item1, item2, item1,
+item2#
+
```

Comment:
All of the code snippets like `#name=item1, ..itemN#` and `item*N` and `item, item, ..., item` need to be enclosed in double-backticks.